app/controllers/documents/intro_controller.rb

```diff
@@ -1,16 +1,11 @@
 module Documents
-  class IntroController < Questions::QuestionsController
+  class IntroController < DocumentUploadQuestionController
```

Comment:
i don't know why this wasn't a document upload controller to begin with, but i made this change because it doesn't require us to define `next_path` anymore, the one on `DocumentUploadQuestionController` works. but maybe i'm overlooking something?